core/src/test/java/org/elasticsearch/index/query/GeoShapeQueryBuilderTests.java

```diff
@@ -55,8 +56,10 @@
 
     @Override
     protected GeoShapeQueryBuilder doCreateTestQueryBuilder() {
-        ShapeBuilder shape = RandomShapeGenerator.createShapeWithin(getRandom(), null);
```

Comment:
thanks a lot for taking care of this, I couldn't figure out why this tests was so slow :)